src/Math/Angle.cs

```diff
@@ -44,7 +41,7 @@ public static Angle Between(Vector2 from, Vector2 to)
         {
             float perpDot = from.Y * to.X - from.X * to.Y;
 
-            return Angle.FromRadians(Mathf.Atan2(perpDot, Vector2.Dot(from, to)));
+            return FromRadians(Mathf.Atan2(perpDot, Vector2.Dot(from, to)));
```

Comment:
I thought we did, especially based on how the math classes look. I guess I am fine with leaving them off, but then we should probable incrementally do that for the rest of the library as well.